src/Runner.Listener/Configuration/CredentialManager.cs

```diff
@@ -40,7 +40,7 @@ public ICredentialProvider GetCredentialProvider(string credType)
             return creds;
         }
 
-        public VssCredentials LoadCredentials()
+        public VssCredentials LoadCredentials(bool preferV2 = true)
```

Comment:
this is used during runner startup, we will prefer the new authorization url from Token service.